### ISSUE TITLE
Object support

### ISSUE THREAD

**Issue by gtfargo** [oblador/react-native-animatable#4]
Allow for multiple transitions on a component

```
transition={{'fontSIze', 'color'}}
transitionValue={{this.state.fontSize, this.state.color}}
```

**Comment by oblador** [comment]
@gtfargo & @tgoldenberg:

I've started a [branch with multiple transitions support](https://github.com/oblador/react-native-animatable/tree/transitionValue-refactor). Check it out and reply back with feedback, if it works well then I'll merge it. 

`transitionValue` is now gone, new syntax is like this:

```html
<Animatable.Text transition={['fontSize', 'color']} style={condition ? styles.someStyle : styles.otherStyle}/>
<Animatable.Text transition="fontSize" style={{fontSize: this.state.fontSize}} />

```

**Comment by gtfargo** [comment]
An example of infinite animations would be shweet.

**Comment by oblador** [comment]
There's already [one](https://github.com/oblador/react-native-animatable#looping)?

**Comment by gtfargo** [comment]
Yup!  I had something twisted on my end that was interfering with the animation.  Sorry for the mixup!

**Comment by oblador** [comment]
This is now merged and released as [0.4.0](https://github.com/oblador/react-native-animatable/releases/tag/v0.4.0).
